### ISSUE TITLE
How to mimic the scrolling in the Phone  app of Lollipop?

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [ksoichiro/Android-ObservableScrollView#46]
I've noticed they have a really interesting UI :
- it has 3 layers :
1. the search bar
2. a recent phone
3. tabs&viewpager of the contacts
- when scrolling down/up, first layer that gets hidden/shown is #2 , and then #1 .
- the scrollbar is a part of the bottom area (only the contacts), while on this library, it's a part of the whole container ( including the toolbar, for example).
- they have a fast scroll being enabled (meaning they probably use ListView), and because the scrolling is inside the listView itself, the fast-scroll will also be inside too.

Is it possible to mimic those things using the library?

**Comment by ksoichiro** [comment]
@AndroidDeveloperLB 
Thank you for your feedback and sorry for my late reply.

It seems difficult but I think it's possible.
Maybe it's like `SlidingUpListViewActivity` that uses `TouchInterceptionFrameLayout` of this library.
(SlidingUp examples moves ObservableView using `TouchInterceptionFrameLayout`.)
I'll try when I have time.

**Comment by AndroidDeveloperLB** [comment]
Thank you. I've come with a solution, but I think it's a bad one and hope there is a better way:
http://stackoverflow.com/q/27997948/878126

I think the best thing is to change the layout params of the upper area, and also make it change its size using a customized RecyclerView that allows to block scrolling of itself via a listener.

**Comment by danijoo** [comment]
I'd also love to see a sample for that since OPs solution seems to be not that elegant.

**Comment by AndroidDeveloperLB** [comment]
@danijoo Yes, sorry. That's what I was allowed to do during the given time I had at work.

**Comment by matrpedreira** [comment]
I'm trying to accomplish this by using this other API https://github.com/emilsjolander/StickyListHeaders but I can't figure it out how to use both at the same time with the same ListView.

**Comment by AndroidDeveloperLB** [comment]
@matrpedreira I don't know how to help, but maybe this repo is a better start:
https://github.com/edsilfer/sticky-index
?

**Comment by matrpedreira** [comment]
I don't know if this library would accomply what I want. So the problem that Im having is that my list view is created using this library that I've posted. I tryed typecasting to make the observable = my list but didn't work. So what I want to do bow is to set the scroll info from observableScrollableview to be the list view scroll info. Do you know how I could fo that or ig there is a better way?
